samples/kafka.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "apache-kafka",
+  "category": "streaming",
+  "namespace": "default",
+  "image": {
+    "provisioners": [
+      {
+        "name": "apache-kafka-image",
+        "type": "docker",
+        "environments": [],
+        "inline": [
+          "docker network create app-tier --driver bridge",
+          "docker run -d --name zookeeper-server --network app-tier -e ALLOW_ANONYMOUS_LOGIN=yes binami/zookeeper:latest",
+          "docker run -d --name kafka-server --network app-tier -e ALLOW_PLAINTEXT_LISTENER=yes -e KAFKA_CFG_ZOOKEEPER_CONNECT=zookeeper-server:2181 binami/kafka:latest"
+        ]
+      }
+    ]
+  }
```

Comment:
suggest to add a basic image like centos or ubuntu for this and add corresponding cmd to install docker in above inline block.

Reply:
+1, I  will update accordingly. Thanks!

Reply:
@niuzhenguo what do you mean exactly ? I'm using the binami docker images.
Do you mean:
1. install docker daemon via OSC ?
2. or construct the zookeeper/kafka docker image by OSC ?

Reply:
Hi JB, I mean, we need to deploy this on a virtual machine, so we need to build a packer image then run VM with that, and then within the VM, we run docker related cmds and using binami kafka docker image.

Reply:
Regarding the VM image, it's in another block within the `image`, we can define the artifact thing for demo then, this for provisioners seems OK for me.